node/hubController.js

```diff
@@ -3,15 +3,17 @@
 'use strict';
 
 var q = require('q');
-var hubsConfig = require("./hubsConfig");
+var HubsConfig = require("./hubsConfig");
 var OpenT2TError = require('opent2t').OpenT2TError;
 var OpenT2TConstants = require('opent2t').OpenT2TConstants;
+var OpenT2TLogger = require('opent2t').Logger;
 
 class HubController {
 
-    constructor() {
+    constructor(logLevel = "info") {
         this.supportedHubsCache = undefined;
         this.OpenT2T = require('opent2t').OpenT2T;
+        this.ConsoleLogger = new OpenT2TLogger(logLevel);
```

Comment:
Yes but for now i wanted the name to be very clear around this logging to console only and we are instantiating opent2tlogger in default console mode only and not specifying a default file transport. (i didnt want to create random files in the cloud environement ) ; I expect the name to change and be more generic once we add support for the custom transport here.

---
In reply to: [103827940](https://github.com/openT2T/cloud/pull/26#discussion_r103827940) [](ancestors = 103827940)